packages/angular_devkit/build_angular/src/builders/application/execute-build.ts

```diff
@@ -154,14 +161,18 @@ export async function executeBuild(
   let indexContentOutputNoCssInlining: string | undefined;
 
   // Generate index HTML file
-  if (indexHtmlOptions) {
+  // If localization is enabled, index generation is handled in the inlining process.
+  // NOTE: Localization with SSR is not currently supported.
```

Comment:
By not supported you mean in this PR, but will be before v17 is fully supported?

Right now, i18n is possible and support when using the server builder, it's just that there is some server configuration/workarounds that users need to do support this for their production server.